rust/headless-client/src/windows.rs

```diff
@@ -8,6 +8,11 @@ use crate::SignalKind;
 use anyhow::Result;
 use std::path::{Path, PathBuf};
 
+pub(crate) const SOCKET_PROTECTOR: () = ();
+
+pub use phoenix_channel::tcp_socket_factory;
```

Comment:
```suggestion
pub(crate) use phoenix_channel::tcp_socket_factory;
```
Same question as for Linux. I don't think the `lib.rs` exports them outside the crate.